docs/contributing/style-guide.md

```diff
@@ -0,0 +1,420 @@
+inline char *health_stock_config_dir(void) {
+    char buffer[FILENAME_MAX + 1];
+    snprintfz(buffer, FILENAME_MAX, "%s/health.d", netdata_configured_stock_config_dir);
+    return config_get(CONFIG_SECTION_HEALTH, "stock health configuration directory", buffer);
+}
+```
+
+### Highlighted lines in code blocks
+
+If you want to direct readers to a specific line within a code block, you can use the `hl_lines` option to highlight as many lines as you would like.
+
+A single line:
+
+````
+```c hl_lines="2"
+inline char *health_stock_config_dir(void) {
+    char buffer[FILENAME_MAX + 1];
+    snprintfz(buffer, FILENAME_MAX, "%s/health.d", netdata_configured_stock_config_dir);
+    return config_get(CONFIG_SECTION_HEALTH, "stock health configuration directory", buffer);
+}
+```
+````
+
+```c hl_lines="2"
+inline char *health_stock_config_dir(void) {
+    char buffer[FILENAME_MAX + 1];
+    snprintfz(buffer, FILENAME_MAX, "%s/health.d", netdata_configured_stock_config_dir);
+    return config_get(CONFIG_SECTION_HEALTH, "stock health configuration directory", buffer);
+}
+```
+
+Or multiple lines:
+
+````
+```c hl_lines="1 2 4"
+inline char *health_stock_config_dir(void) {
+    char buffer[FILENAME_MAX + 1];
+    snprintfz(buffer, FILENAME_MAX, "%s/health.d", netdata_configured_stock_config_dir);
+    return config_get(CONFIG_SECTION_HEALTH, "stock health configuration directory", buffer);
+}
+```
+````
+
+```c hl_lines="1 2 4"
+inline char *health_stock_config_dir(void) {
+    char buffer[FILENAME_MAX + 1];
+    snprintfz(buffer, FILENAME_MAX, "%s/health.d", netdata_configured_stock_config_dir);
+    return config_get(CONFIG_SECTION_HEALTH, "stock health configuration directory", buffer);
+}
+```
+
+### Combining all the highlighting functions
+
+Let's get a little wild!
+
+```c linenums="36" hl_lines="1 2 4"
+inline char *health_stock_config_dir(void) {
+    char buffer[FILENAME_MAX + 1];
+    snprintfz(buffer, FILENAME_MAX, "%s/health.d", netdata_configured_stock_config_dir);
+    return config_get(CONFIG_SECTION_HEALTH, "stock health configuration directory", buffer);
+}
+```
+
+## Accessibility
+
+Netdata's documentation should be as accessible as possible to as many people as possible. While the rules about [tone and content](#tone-and-content) and [language and grammar](#language-and-grammar) are helpful to an extent, we also need some additional rules to improve the reading experience for all readers.
+
+
+### Images
+
+Provide alt text for every image you include in Netdata's documentation. It should summarize the intent and content of the image.
+
+Don't use images of text, code samples, or terminal output. Instead, put that text content in a code block so that all devices can render it clearly and screen readers can parse it.
+
+Don't rely on images to present new information, such as directions on where to click a button or UI element. Be sure to explain each image with text in case a reader can't see images for whatever reason.
```

Comment:
I understand why you're saying this, but people generally avoid adding images. They only include them when it's very difficult to explain with words what needs to happen. I don't think the instruction needs to change, just noting that we may be asking too much.